finding_data.md

```diff
@@ -19,6 +19,46 @@ You can change the order of the results using the 'Sort by' drop-down.
 
 ![Sort dropdown](images/finding_data_sort.png)
 
+### Keyword search tips 
+
+When using keyword search, you can use advanced operators to enhance your query.
+The examples below show how you can use those operators.
+
+* ```"armed forces" expenses``` - search for exact phrases by putting quote marks around them
+
+* ```primary school -care age``` - exclude words or quoted phrases by using a minus sign or NOT
+
+* ```primary OR secondary school``` - use 'OR' to allow alternatives (the default is to 'AND' all of the terms)
+
+You can also search specific fields, ```publisher:cabinet-office res_format:CSV``` searches only for CSV files published by the Cabinet Office.
+
+Some of the fields that are available to you are listed below:
+
+* __name__ - Find datasets by matching words in the name of the dataset.
+* __title__ - Match words in the title of the dataset
```

Comment:
Seems repetitive always saying "Match words in the" or "Find by matching words in" or "Filter by" or "Show only" for every field. Just say what the field represents.